### DISCUSSION TITLE
Jerky Search (maybe with incsearch)

### DISCUSSION THREAD

**Discussion by canmetan** [JetBrains/ideavim#547] [Q&A]
Before I write anything, questions [this](https://github.com/JetBrains/ideavim/discussions/520) and [this](https://github.com/JetBrains/ideavim/discussions/536) do not answer my question I think.

When searching for text with "/" in ideavim, the interface is jarringly jerky for me.

Explanation with GIFs:

- When searching in VIM, I press "/" and type "one way to" and then press enter, the interface remains in the same position (expected behavior):
![vim](https://user-images.githubusercontent.com/8360917/190946680-685c7eb9-31cb-4886-a112-21d73815716d.gif)

- When searching in ideavim, I follow the same process by typing "/one way to" and then pressing Enter somehow jump the view up and then back onto the place that I have searched for. It's a few millisecond jump up and back (hopefully you can see from the gifs; I think it jumps to the start of the search and back at the caret of search)

![intellij](https://user-images.githubusercontent.com/8360917/190946689-0211804b-0bdc-4ea3-aa58-b6c4f4ce0c15.gif)

I find this to be quite jarring and would love to know how to fix it.
In my vimrc I have `set incsearch` and this is my .ideavimrc 

```
source ~/.vimrc

" Be able to comment out code
sethandler <C-/> a:ide
" Make the icon gray
set ideajoin
set ideastatusicon=gray

```

**Comment by AlexPl292** [comment]
Hi, thank you for the report. I was able to reproduce it. I'll take a look.

**Reply by canmetan** [reply]
Any luck on this issue?

**Reply by citizenmatt** [reply]
This is an implementation detail of how `incsearch` works, unfortunately. Here's an issue to track and vote for: [VIM-2529](https://youtrack.jetbrains.com/issue/VIM-2529/IdeaVim-Ugly-jump-with-set-incsearch-turned-on).
